Formula/povray.rb

```diff
@@ -16,46 +15,38 @@ class Povray < Formula
   depends_on :macos => :lion
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  # Reported upstream 2017-09-02
+  # https://github.com/POV-Ray/povray/issues/317
+  depends_on "boost@1.60"
```

Comment:
Would you accept a patch to remove the wrong header includes, instead of depending on an old boost?

Reply:
if it gets submitted upstream :)

Reply:
Here's https://github.com/POV-Ray/povray/pull/318 adapted for stable:
https://github.com/Homebrew/formula-patches/pull/173

Reply:
@JCount note that it also means `ENV.cxx11` needs to be added.